controllers/extension-certificate-service/pkg/apis/config/v1alpha1/register.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// GroupName is the group name use in this package
+const GroupName = "certificate-service.extensions.gardener.cloud"
```

Comment:
```suggestion
const GroupName = "config.certificate-service.extensions.gardener.cloud"
```

by convention this group is contains `config.`:
- https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/apis/config/register.go#L25
- https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/config/register.go#L25
- https://github.com/gardener/gardener/blob/master/pkg/controllermanager/apis/config/register.go#L23

Reply:
Okay, but to follow the convention shouldn't it rather be `certificate-service.extensions.config.gardener.cloud` ?

Reply:
Not every component has a config API. The scope of the cert-service extension is `certificate-service.extensions.gardener.cloud`, so I guess the cleanest way is to add `config` to the beginning (I'm doing it similarly with the worker controller: `config.aws.provider.extensions.gardener.cloud`).

Reply:
As discussed, we're going to go with `certificate-service.extensions.config.gardener.cloud` since we already use `config.gardener.cloud` for the `Gardener-Controller-Manager` and it is aligned with the config structure of k8s.io.